src/App.jsx

```diff
@@ -1,39 +1,47 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-import './App.scss';
+// import './App.scss';
 
-// Why is this source of truth beneficial when spread across a global organization?
 import Header from './Components/Header';
 import Footer from './Components/Footer';
 import Form from './Components/Form';
 import Results from './Components/Results';
 
-const App = () => {
+function App () {
 
   const [data, setData] = useState(null);
   const [reqParams, setReqParams] = useState ({});
   const [loading, setLoading] = useState(false);
 
+  useEffect(() => {
+    console.log('do we have an api')
+    async function getData(){
+      console.log(getData)
+      const response = await axios.get(`https://pokeapi.co/api/v2/${reqParams.url}`);
+     console.log('api response: ', response.data);
+     setData(response.data);
+    } 
+    if (reqParams.url && reqParams.method) {
+      getData();
+    }
+  }, [reqParams]);
+
+
+
   const callApi = (reqParams) => {
     setLoading(true);
     setTimeout (() => {
-      const data = {
-        count: 2,
-        results: [
-        {name: 'fake thing 1', url: 'http://fakethings.com/1'},
-        {name: 'fake thing 2', url: 'http://fakethings.com/2'},
-      ],
-      };
-      setData(data);
       setReqParams(reqParams);
+      setLoading(false);
     }, 1000);
   };
```

Comment:
Now that we're not spoofing data, we probably want to get rid of the `setTimeout` and move our `setLoading` so that it changes back to false when our data is actually done loading.